src/PrestaShopBundle/Controller/Admin/CommonController.php

```diff
@@ -118,7 +118,7 @@ public function paginationAction(Request $request, $limit = 10, $offset = 0, $to
                 unset($callerParameters[$k]);
             }
         }
-        $routeName = $request->attributes->get('caller_route', $request->attributes->get('caller_parameters', ['_route' => false])['_route']);
```

Comment:
Are you sure it's not a BC break?

Reply:
I've rechecked, and seems that product page is using it too, so ill fall back to passing the `caller_route` and `caller_parameters` in twig where that controller is being rendered, it should be enough to avoid possible bc break, thanks